sycl/doc/extensions/LevelZeroBackend/LevelZeroBackend.md

```diff
@@ -293,6 +312,25 @@ should take ownership of the passed native handle. The default behavior is to
 transfer the ownership to the SYCL runtime. See section 4.4 for details. If
 the behavior is "transfer" then the runtime is going to destroy the input
 Level-Zero kernel</td>
+</tr><tr>
+<td>
+
+``` C++
+template <backend Backend, typename T, int Dimensions = 1,
+          typename AllocatorT = buffer_allocator>
+typename std::enable_if<Backend == backend::ext_oneapi_level_zero,
+                        buffer<T, Dimensions, AllocatorT>>::type
+make_buffer(
+    const backend_input_t<backend::ext_oneapi_level_zero,
+                          buffer<T, Dimensions, AllocatorT>> &BackendObject,
+    const context &TargetContext, event AvailableEvent = {})
+```
+</td>

```

Comment:
The Level Zero backend interop extension is considered "supported" now, so we need to be more careful about changes that add new APIs.  This new API should have a matching change to the feature-test macro (`SYCL_EXT_ONEAPI_BACKEND_LEVEL_ZERO `) that is described in the Introduction section.  Specifically:

* Add a new row to the table in the Introduction section for version 2 of this macro.  The new row should state that support was added for the `make_buffer()` API.

* The description of `make_buffer()` should clearly state that it is available starting with revision 2 of this specification.

Reply:
Thank you, fixed.